lock-impl/src/test/java/com/palantir/lock/impl/ClientAwareLockTest.java

```diff
@@ -48,8 +47,7 @@
 @ShouldRetry
 public final class ClientAwareLockTest {
 
-    private static final ExecutorService executor = PTExecutors.newCachedThreadPool(
-            new NamedThreadFactory(ClientAwareLockTest.class.getName(), true));
+    private static final ExecutorService executor = PTExecutors.newCachedThreadPool("ClientAwareLockTest");
```

Comment:
nit: Don't feel strongly about which way this falls, but we should probably be consistent on whether we prefer the string literal or `ThisClass.class.getName()` for these executors.

Reply:
that's fair, I'll revert these out to use class.getName(), no reason to churn on these in this change.